contracts/modules/DToken.sol

```diff
@@ -53,7 +54,7 @@ contract DToken is BaseLogic {
         return assetCache.totalBorrows / INTERNAL_DEBT_PRECISION / assetCache.underlyingDecimalsScaler;
     }
 
-    /// @notice Sum of all outstanding debts, in underlying units with extra precision (increases as interest is accrued)
+    /// @notice Sum of all outstanding debts, in underlying units always normalized to 18 (increases as interest is accrued)
```

Comment:
This also includes `INTERNAL_DEBT_PRECISION` right? If so I think it's actually normalised to 27 decimals.

Reply:
fixed